src/assisted_test_infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -219,27 +213,17 @@ def format_node_disk(self, node_name: str, disk_index: int = 0):
         log.info("Formatting disk for %s", node_name)
         self.format_disk(f"{self.params.libvirt_storage_pool_path}/{self.entity_name}/{node_name}-disk-{disk_index}")
 
-    def get_ingress_and_api_vips(self, is_highly_available: bool = True) -> Dict[str, str]:
-        """Get appropriate values for setting static access endpoint IPs.
-
-        For multi-node clusters, we're using <sub-net>.100 for the API endpoint and <sub-net>.101
-        for the ingress endpoint (the IPv6 values are appropriately <sub-net>:64 and <sub-net>:65).
+    def get_ingress_and_api_vips(self) -> Dict[str, str]:
+        """Pick two IPs for setting static access endpoint IPs.
 
-        For single-node clusters ingress endpoint will be the same as the API endpoint (and it will
-        follow the same values as above for IPv4 and IPv6).
+        Using <sub-net>.100 for the API endpoint and <sub-net>.101 for the ingress endpoint
+        (the IPv6 values are appropriately <sub-net>:64 and <sub-net>:65).
 
-        Args:
-            is_highly_available (bool): whether IPs should be adapted to highly available clusters or not.
+        This method is not applicable for SNO clusters, where access IPs should be the node's IP.
         """
-        network_subnet_starting_ip = str(
-            ipaddress.ip_address(ipaddress.ip_network(self.get_primary_machine_cidr()).network_address) + 100
-        )
-        ips = utils.create_ip_address_list(2, starting_ip_addr=str(ipaddress.ip_address(network_subnet_starting_ip)))
-
-        if is_highly_available:
-            return {"api_vip": ips[0], "ingress_vip": ips[1]}
-
-        return {"api_vip": ips[0], "ingress_vip": ips[0]}
+        network_subnet_starting_ip = ip_address(ip_network(self.get_primary_machine_cidr()).network_address)
```

Comment:
Initially I found this change confusing, but then I realised that the +100 has been moved to the line below, I think it looks clearer that way.

We discussed this offline, just putting my thoughts here so we can record them.

IMO this looks good.